DigitalLearningSolutions.Web/Controllers/TrackingSystem/Centre/Administrator/AdministratorController.cs

```diff
@@ -115,14 +116,18 @@ public IActionResult EditAdminRoles(EditRolesViewModel model, int adminId)
                 return NotFound();
             }
 
-            userService.UpdateAdminUserPermissions(
-                adminId,
+            var adminRoles = new AdminRoles(
                 model.IsCentreAdmin,
                 model.IsSupervisor,
-                model.IsTrainer,
                 model.IsContentCreator,
+                model.IsTrainer,
                 model.ContentManagementRole.IsContentManager,
-                model.ContentManagementRole.ImportOnly,
+                model.ContentManagementRole.ImportOnly
+            );

```

Comment:
Could we just add this as a method on the ViewModel so:
1. We can write a simple unit test for it.
2. We can just do model.GetAdminRoles() when calling UpdateAdminUserPermissions below?